test/user_test.rb

```diff
@@ -64,4 +64,32 @@ def test_user_has_organizers
       assert_kind_of Chiketto::Organizer, @user.organizers.first
     end
   end
+
+  def test_user_has_contact_lists
+    find_me
+
+    VCR.use_cassette 'user-contact-lists' do
+      assert_kind_of Chiketto::ContactList, @user.contact_lists.first
+    end
+  end
+
+  def test_user_find_contact_list_returns_contact_list
+    find_me
+
+    VCR.use_cassette 'user-find-contact-list' do
+      assert_kind_of Chiketto::ContactList, @user.find_contact_list(411987)
```

Comment:
It's probably worthwhile to extract the contact_list_id out to a constant up with USER_ID since it will need to be changed by anyone running the tests to match an id available to their api key.